src/postprocessors/RMSDistance.C

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************/
+/*                     DO NOT MODIFY THIS HEADER                      */
+/* MAGPIE - Mesoscale Atomistic Glue Program for Integrated Execution */
+/*                                                                    */
+/*            Copyright 2017 Battelle Energy Alliance, LLC            */
+/*                        ALL RIGHTS RESERVED                         */
+/**********************************************************************/
+
+#include "RMSDistance.h"
+
+registerMooseObject("MooseApp", RMSDistance);
+
+template <>
+InputParameters
+validParams<RMSDistance>()
+{
+  InputParameters params = validParams<ElementIntegralVariablePostprocessor>();
+  params.addRequiredParam<Point>("point", "Point to which the RMS distance is computed");
+  params.addClassDescription("Computes the RMS distance of a distribution with respect to a fixed point.\n"
+                             "If r0 is the fixed point and variable is denoted by f then"
+                             " RMS = sqrt( integral{r^2 f} / integral{f}) ");
```

Comment:
I wouldn't put the equation here, they appear twice in the docs now and only once properly rendered.